docs/community/meetings/2024-05-09.md

```diff
@@ -0,0 +1,201 @@
+
+Conceptually yes, but this structure also involved a lot of trade-offs — even the sheer volume of data that had to be saved by one model and read into another.
+It was difficult to pick up and use the code **as a whole**.
+And there were related issues, such as being able to successfully install [greta](https://greta-stats.org/).
+
+> Nick Tierney: I know that greta can be difficult to install, and I can help people with this.
+
+T.K. also noted that they found some minor inconsistencies between the models, such as whether a date was used to identify the start or end of its 24-hour interval.
+
+## Tools and platforms
+
+!!! tip "Key message"
+
+    Personal preferences can play a major role in deciding which tools are best for a project.
+
+The various models were hosted in repositories on [Bitbucket](https://bitbucket.org/), [GitHub](https://github.com/), and the University of Melbourne's [GitLab instance](https://gitlab.unimelb.edu.au/).
+T.K. noted that the only discernible differences between these platforms was how they handled authorisation and authentication.
+
+T.K. also explored several different editors, some which were language-specific:
+
+- VS Code was the easiest for looking at, and searching through, the code;
+- RStudio was the worst; and
+- Matlab was somewhat hard to use, but T.K. acknowledged they were also the least familiar with this language.
+  They also experienced issues running Matlab on high-performance computing platforms.
+
+T.K. noted they had previous experience with Eclipse (an IDE for Java) and Visual Studio (which felt very "heavy").
+
+**Question:** what were your favourite things in VS Code, and what made RStudio the worst?
+
+It was easiest to open a project in VS Code, RStudio would always open up an entire project or previous workspace, rather than just opening a single file.
+RStudio also kept asking to update itself.
+
+T.K.  also strongly disliked the RStudio font, which was another source of friction.
+They tried installing an RStudio extension for VS Code, but weren't sure how well it worked.
+
+> Nick Tierney: R history is really annoying, but you can turn it off.
+> I'm not sure why it's enabled by default, all of the RStudio recommendations involve turning off these kinds of things.
+
+> Rahmat Sagara: I'm potentially interested in using VS Code instead of RStudio.
+
+> Eamon Conway: the worst thing about VS code is that debugging is very hard to setup.
+
+## Task management
+
+!!! tip "Key message"
+
+    Task management is hard, and switching to a new system during a project is extremely difficult.
+
+T.K. reported trying to use Gitlab issues to plan out what to do and how to do it, but found they weren't a good fit with their workflow.
+They then trialled [Trello boards](https://trello.com/) for different versions, but stopped using them due to a lack of time to manage the boards.
+In review:
+
+- T.K. uses [TickTick](https://www.ticktick.com/) for personal stuff, but it's ugly!
```

Comment:
I use Todoist for personal tasks (and I like it); I use TickTick for work tasks (and it's ugly)

Reply:
Thanks, my note-taking clearly couldn't keep up with the conversation. Done.